decidim-core/lib/decidim/api/interfaces/followable_interface.rb

```diff
@@ -7,7 +7,12 @@ module FollowableInterface
       include Decidim::Api::Types::BaseInterface
       description "An interface that can be used in followable objects."
 
+      field :followers, [Decidim::Core::AuthorInterface, { null: true }], "The followers of this object.", null: false
```

Comment:
Resource or object? With the final dot or not? 

```suggestion
      field :followers, [Decidim::Core::AuthorInterface, { null: true }], "The followers of this resource", null: false
```